lib/mempool.go

```diff
@@ -1954,6 +1954,18 @@ func ComputeTransactionMetadata(txn *MsgDeSoTxn, utxoView *UtxoView, blockHash *
 			Metadata:             "NewMessageRecipientAccessGroupOwnerPublicKe",
 		})
 	}
+	// Check if the transactor is an affected public key. If not, add them.
+	if txnMeta.TransactorPublicKeyBase58Check != "" {
+		for _, affectedPublicKey := range txnMeta.AffectedPublicKeys {
+			if affectedPublicKey.PublicKeyBase58Check == txnMeta.TransactorPublicKeyBase58Check {
+				return txnMeta
```

Comment:
I don't love the return here, since if we add more code at the end of this later it'll cause a subtle bug. I'd just have an isFound bool and then do `if !isFound` below.